tests/modules/encounters/resources/test_modify_encounter.py

```diff
@@ -159,14 +157,13 @@ def edm_return_500(*args, **kwargs):
     with mock.patch.object(
         flask_app.edm, 'request_passthrough', side_effect=edm_return_500
     ):
-        res = enc_utils.patch_encounter(
+        enc_utils.patch_encounter(
```

Comment:
Yeah, not sure I'm completely on board with removing this type of doublecheck. Not going to worry about it too much though.